### PR TITLE
platform: Set the integration-auth as NONE and disable the metrics in default

### PR DESCRIPTION
**Pull request by lijamie98**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [x] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [x] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What

The default value of integration-auth configuration is JWT_TOKEN. We should set it to none.

### Why

N/A

### Known limitations

N/A

### CODE REVIEW

**Comment by reecexlm** [comment]
can you also make default for metrics service disabled in this PR as well.

File: anchor-reference-server/src/main/java/org/stellar/anchor/reference/config/IntegrationAuthSettings.java [hunk 1]
```diff
@@ -1,13 +1,11 @@
 package org.stellar.anchor.reference.config;
 
 import lombok.Data;
-import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 import org.stellar.anchor.config.IntegrationAuthConfig.AuthType;
 
 @Data
 @Configuration
-@ConfigurationProperties(prefix = "integration-auth")
```
Comment (marcelosalloum) [inline anchor-reference-server/src/main/java/org/stellar/anchor/reference/config/IntegrationAuthSettings.java:10]: Why are you removing this? Won't it be impossible for the reference server to pick up the configuration?

Comment (lijamie98) [reply anchor-reference-server/src/main/java/org/stellar/anchor/reference/config/IntegrationAuthSettings.java]: That's right. I have added this back.

File: platform/src/main/resources/anchor-config-defaults.yaml [hunk 2]
```diff
@@ -86,16 +86,16 @@ app-config:
     #     NONE: no authentication is used
     #     API_KEY: the authentication is done using an API key added to the `X-Api-Key` header.
     #     JWT_TOKEN: the authentication is done using a JWT token added to the `Authorization` header. this token is generated from the secret key.
-    authType: JWT_TOKEN
+    authType: NONE
```
Comment (marcelosalloum) [inline platform/src/main/resources/anchor-config-defaults.yaml:89]: The default value for the reference server is still JWT_TOKEN though:
https://github.com/stellar/java-stellar-anchor-sdk/blob/702bbb66cf44491cc0af39223abb108a8e98e44f/anchor-reference-server/src/main/resources/anchor-reference-server.yaml#L26

**Review by marcelosalloum — APPROVED** [review]

File: platform/src/main/resources/anchor-config-defaults.yaml (hunk 2 above)
Comment (lijamie98) [reply platform/src/main/resources/anchor-config-defaults.yaml:89]: Fixed!

**Pull request merged**